release_notes/Service_Fabric_ReleaseNotes_82CU2.md

```diff
@@ -0,0 +1,79 @@
+ If the application is stuck in rollback, the rollback has to be first completed before the application can be upgraded again.
+* **.NET Core 2.x Support**: .NET Core runtime LTS 2.1 runtime is out of support as of Aug 21, 2021, and .NET Core runtime 2.2 is out of support from Dec 2019. Service Fabric releases after those dates will drop support for Service Fabric apps running with .NET Core 2.1 & .NET Core 2.2 runtimes respectively. Current apps running on .NET Core 2.x runtime will continue to work, but requests for investigations or request for changes will no longer receive support. Service Fabric .NET SDK will take a dependency on .NET runtime 3.x features to support Service Fabric .NET Core apps.
+* **Ubuntu 16.04 Support**: Ubuntu 16.04 LTS reached it's 5-year end-of-life window on April 30, 2021. Due to this, Service Fabric runtime has dropped support for 16.04 LTS and we recommend moving your clusters and applications to Ubuntu 18.04 LTS. Current applications running on it will continue to work, but requests for investigations or requests for change will no longer receive support.
+* **ASP.NET Core 1.0 Support**: Service Fabric ASP.NET Core packages are built against ASP.NET Core 1.0 binaries which are out of support. Starting Service Fabric 8.2, we will be building Service Fabric ASP.NET Core packages against .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, .NET Framework 4.6.1. As a result, they can be used only to build services targeting .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, >=.NET Framework 4.6.1 respectively.<br>For .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, Service Fabric ASP.NET Core will be taking dependency on Microsoft.AspNetCore.App shared framework, whereas for NetFx target frameworks >=.NET Framework 4.6.1, Service Fabric ASP.NET Core will be taking dependency on ASP.NET Core 2.1 packages.<br>The package Microsoft.ServiceFabric.AspNetCore.WebListener will no longer be shipped, Microsoft.ServiceFabric.AspNetCore.HttpSys package should be used instead.
+* **Windows Server 2022**: Service Fabric runtime will officially provide support for Windows Server 2022 starting with Service Fabric 8.1 CU3 release and all future releases.
+
+
+## Service Fabric Common Bug Fixes
+
+ | Versions | IssueType | Description | Resolution | 
+ | - | - | - | - | 
+ | **Windows - 8.2.1486.9590<br>Ubuntu 18 - 8.2.1285.1** | **Bug** | Disable FabricClient Cache in HTTP Gateway | **Brief Description**: Http gateway cache inside the FabricGateway.exe process does not automatically flush stale endpoint data from cache. As such, clients may end up receiving stale endpoints. <br> **Impact**: Client code calling http gateway can receive stale endpoints. <br> **Fix**: Disable the client cache inside http gateway. This forces http gateway to always query for the most recent data. <br> **Workaround**: A possible workaround is manually restarting the FabricGateway.exe process that is returning stale data. | 
```

Comment:
* Standardize on the capitalization of http

Reply:
Done!